ci.sh

```diff
@@ -12,19 +12,19 @@ function build_rust_targets() {
   cargo build --release --manifest-path crypto/Cargo.toml --no-default-features
   cargo build --release --manifest-path platform/Cargo.toml --no-default-features
   cargo build --release --manifest-path dpe/Cargo.toml --features=$profile --no-default-features
-  cargo build --release --manifest-path simulator/Cargo.toml --features=$profile,openssl --no-default-features
+  cargo build --release --manifest-path simulator/Cargo.toml --features=$profile,rustcrypto --no-default-features
```

Comment:
should we just drop this feature and assume it's always set?

Reply:
ok to do in a separate pr

Reply:
I'll follow up - I think this will be a little tricky to do